tests/test_hypersphere.py

```diff
@@ -400,6 +400,34 @@ def test_mean_and_belongs(self):
         result = self.metric.mean([point_a, point_b, point_c])
         self.assertTrue(self.space.belongs(result))
 
+    def test_diameter(self):
+        n_points = 10
+        points = self.space.random_uniform(n_points)
+        result = self.metric.diameter(points)
+        gs.testing.assert_allclose(result.size, 1)
```

Comment:
Could you also test the value of the result (not only its size) with a handmade set of points (maybe 3 points?). Thank you.